app/views/search/_sitetour_modal.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade sitetour" id="sitetourModal" style="font-family: FontAwesome;" tabindex="-1" role="dialog">
```

Comment:
Why are you setting inline styles here (font-family)? We should be setting things like that in a sass file. Also, the modal text is not inheriting the global font styles. Notice that all fonts throughout the app are sans serif while the font in the modal is serif.